### DISCUSSION TITLE
希望可以添加文件的访问次数和下载次数统计

### DISCUSSION THREAD

**Discussion by 1592363624** [AlistGo/alist#3274] [Ideas]
希望可以添加文件的访问次数和下载次数统计

**Comment by jumoon2023** [comment]
建议增加个账户登入统计倒是有必要，记录访问账号，访问时间，访问来源ip。用于判单分享出去的账号有没有被滥用。

**Reply by luosxn** [reply]
如果需要访问统计的话可以在头部插入第三方的统计代码

**Comment by 634287319** [comment]
很有必要，顶一下！i need this

**Comment by 1273082756** [comment]
同求, 顶一下

**Comment by diyanqi** [comment]
同求

**Comment by tangseng110** [comment]
github没众筹功能吗？有想要此功能的直接付款，让有能力的大神添加

**Reply by xin2233** [reply]
alist有付费版本，按照你的想法，在付费版本里增加这些新特性即可。
